runbot_gitlab_minimal/controllers/gitlab_ci_controller.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Copyright <2017> <Vauxoo info@vauxoo.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+
+from openerp import http, SUPERUSER_ID
+from openerp.http import request
+from openerp.addons.runbot import runbot
+
+_logger = logging.getLogger(__name__)
+
+
+class RunbotCIController(runbot.RunbotController):
+
+    @http.route(['/runbot/hook_gitlab/<int:repo_id>',
```

Comment:
@moylop260 
The parent method https://github.com/Vauxoo/odoo-extra/blob/60523f7b1be542aa15a7cbd838cc108b002e97bd/runbot/runbot.py#L1247 and this method have diferent url
Parent method 
`@http.route(['/runbot/hook/<int:repo_id>', '/runbot/hook/org'] ...`
This method
`@http.route(['/runbot/hook_gitlab/<int:repo_id>', '/runbot/hook_gitlab/org'] ...`